snuba/subscriptions/executor_consumer.py

```diff
@@ -188,6 +190,8 @@ def __init__(
         max_concurrent_queries: int,
         metrics: MetricsBackend,
         next_step: ProcessingStrategy[SubscriptionTaskResult],
+        # TODO: To be removed once executor is fully rolled out
```

Comment:
Could you provide some additional context around why the commit would be removed when the executor is rolled out

Reply:
I added an additional comment. This comment will be removed pretty soon along with the argument anyway.